.github/workflows/clair-in-ci.db.yaml

```diff
@@ -29,14 +29,14 @@ jobs:
           fi
 
       - name: Log into registry ${{ env.REGISTRY }}
-        uses: docker/login-action@v2.2.0
+        uses: redhat-actions/podman-login@v1
```

Comment:
This is the one piece of this I haven't been able to test.  This action is hosted at github.com/redhat-actions/podman-login.  Will github just automatically clone the repo to use the action or are there some additional steps I need to take to make sure this works properly?

Reply:
I'd just trust it